.github/workflows/terragrunt-deploy.yml

```diff
@@ -93,6 +93,21 @@ jobs:
             echo "TARGET_LIQUIBASE_TAG=--tag=${{inputs.TARGET_LIQUIBASE_TAG}}" >> $GITHUB_OUTPUT
           fi
 
+      - name: Get digest of docker image
+        id: getDigest
+        run: |
+          export IMAGE='ghcr.io/${{github.repository_owner}}/${{ steps.updateImageName.outputs.IMAGE_NAME}}:${{ inputs.TAG }}'
```

Comment:
I don't think you have updateImageName action - this should be instead the same as what you have on line 130 in the main branch

Reply:
@vivid-cpreston updated